packages/test/src/test_setup.js

```diff
@@ -4,6 +4,7 @@ import 'regenerator-runtime/runtime';
 const { configure } = require('enzyme');
 const Adapter = require('enzyme-adapter-react-16');
 
+jest.unmock('@theforeman/test');
```

Comment:
This is fixing the issue so we can run `npm run foreman-js:link && npm test` in foreman/plugin